community/kernel/src/test/java/org/neo4j/kernel/impl/api/integrationtest/ConstraintsCreationIT.java

```diff
@@ -309,28 +493,116 @@ public void shouldDropConstraintIndexWhenDroppingConstraint() throws Exception
     @Test
     public void shouldNotDropConstraintThatDoesNotExist() throws Exception
     {
-        // when
+        for ( PropertyConstraint constraint : new PropertyConstraint[]{
```

Comment:
In a way yes. In another way it is nice to have these tests spelled out.

My thought was to parameterize it through a test base class containing the shared tests, and then have an abstract `createConstraint()` method that each actual test class can implement along with the tests that apply only to that constraint.